CMakeLists.txt

```
@@ -36,27 +36,28 @@ else()
 endif()
 
 set(
-    AGE_CORE_INCLUDE_DIRECTORIES
-    ${CMAKE_SOURCE_DIR}/dependencies/cds/include
-    ${CMAKE_SOURCE_DIR}/src/core
+        AGE_CORE_INCLUDE_DIRECTORIES
+        ${CMAKE_SOURCE_DIR}/dependencies/cds/include
+        ${CMAKE_SOURCE_DIR}/src/core
```

I know this is a pain - 2 spaces in CMakeLists.txt as well. Adjust your auto code formatting in your IDE to match this. 
```suggestion
    AGE_CORE_INCLUDE_DIRECTORIES
    ${CMAKE_SOURCE_DIR}/dependencies/cds/include
    ${CMAKE_SOURCE_DIR}/src/core
```